src/v2/Apps/Conversation/Components/Reply.tsx

```diff
@@ -154,6 +154,7 @@ export const Reply: React.FC<ReplyProps> = props => {
           conversation={conversation}
           openInquiryModal={() => openInquiryModal()}
           openOrderModal={() => openOrderModal()}
+          scrollMessageContainer={onScroll}
```

Comment:
So this is the `onScroll` callback from some other component that gets called on mount in this component? The naming here is a little unclear to me — I would assume a container to be a component of some sort, not a callback. Can this be `onMount` or something similar?

Reply:
well, the thing is we have scrollToBottom handler in a parent component, and I have to throw it via several nodes. And the Reply component has the onScroll={scrollToBottom} prop, and use it for scrolling the message container on updating the chat after sending a message, not for execute it on mounting. So I decided not to create other prop with the same link to the same handler but use the existing onScroll prop.

Reply:
Conversation -> Reply (uses onScroll when message is sent and updated) -> ... -> MakeOfferButton (has to use onScroll for executing it on didMount)

Reply:
`
        <Reply
          onScroll={scrollToBottom}
          onMount={scrollToBottom}
        />
`
Do you want me to do like that?

Reply:
Whatever makes sense to you — but in general I think passing any callbacks using the `onFoobar` naming convention is desirable.

Reply:
Refactored the naming issue